scripts/runner.py

```diff
@@ -51,6 +52,8 @@ def log_data(refresh_rate, logger_list, client):
     logging.info("DB Client Configured")
     farm_name = farm.get_name()
 
+    recipe_list = load_grow(farm, "ceaos.resources.config", "config_lettuce_grow.yml")
```

Comment:
This is a perfectly valid paradigm for loading grow recipes, but what happens when we have multiple, or different grow recipies for different beds or environments?

Reply:
not sure how to go about this

Reply:
You should move the recipes to a subfolder in config, and just generate a list of the recipes. In the config file, you can parse the references to the recipes, and apply them after/during the config loading process.